forc/src/cli/mod.rs

```diff
@@ -32,6 +32,7 @@ enum Forc {
     #[clap(name = "addr2line")]
     Addr2Line(Addr2LineCommand),
     Build(BuildCommand),
+    B(BuildCommand),
```

Comment:
Hmmmmm. Rather than this, I think `clap` allows you to specify a short form?

Reply:
Yeah I'm currently looking for this, but haven't been able to find it:

https://github.com/clap-rs/clap/blob/v3.1.9/examples/derive_ref/README.md#command-attributes

I wonder if it's only available on arguments and not subcommands?

Reply:
Ah good point!

Reply:
One thing it might be worth adding to these short-hand commands is a `#[clap(about = "...")]` section that mentions the command is simply short-hand for the longer form. Otherwise I think it might generate duplicate "about" text under `forc --help`.

Reply:
I've opened https://github.com/clap-rs/clap/issues/3639.

Reply:
Can you file an issue in this repo to track improving the changes in this PR once clap merges that issue?

Reply:
Looks like there's a solution already and we just didn't read deep enough!

https://github.com/clap-rs/clap/issues/3639#issuecomment-1101350736

@amadeusluntz would you mind applying the recommendation mentioned in the above linked comment? That is, rather than adding a separate `B` enum variant, add the following attribute to `Build`:

```rust
    #[clap(alias = "b")]
    Build(BuildCommand),
```

that said the comment's recommendation looks a little different to the docs. If the above doesn't work, maybe try:

```rust
    #[clap(alias("b"))]
    Build(BuildCommand),
```

Reply:
Oh and the same goes for `Test` too.

Reply:
My bad, I just tested long/short, was unaware of `alias` working for enums. Changed to `visible_alias`.